vpr/src/base/vpr_context.h

```diff
@@ -559,6 +561,11 @@ struct RoutingContext : public Context {
 
     /** Is flat routing enabled? */
     bool is_flat;
+
+    /// @brief Post-placement estimate of CHANX routing utilization per (layer, x, y) location.
```

Comment:
We could minimize the lifetime of these variables; we can load them, update the accumulated costs and then destroy them.

Reply:
Test with --route (only) to see this works.  We might have a flow issue where you have to go through placement to run routing.  We could have the router call the placement bounding box etc. code instead in order to localize the creation of these matrices.

Reply:
If you leave the lifetime of these as "placer creates, router consumes" I think that should be commented here.  But I'm not crazy about it if there is an alternative ...